content/dstu2/immunization.md

```diff
@@ -20,9 +20,12 @@ Search for Immunizations that meet supplied query parameters:
 ### Parameters
 
  Name      | Required? | Type          | Description
------------|-----------|---------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+-----------|-----------|---------------|-----------------------------------------------------------------------------------------------------
  `patient` | Y         | [`reference`] | The patient for the vaccination record. Example: `12345`
- `date`    | N         | [`date`]      | Date range into which the immunization administration date falls. Must be prefixed by 'ge' or 'le'. Single dates are treated as implied ranges. EG: `date=ge2014-09-24` `&date=le2015-09-24`
+ `date`    | N         | [`date`]      | Date range into which the immunization administration date falls. Must be prefixed by 'ge' or 'le'.
+
+Notes:
+ - The date parameter value should be prefixed once by 'ge' representing the earliest date, and once by 'le' representing the latest date. Examples: 'date=ge2015-01-01&date=le2016-01-01', 'date=ge2015-01-01', 'date=le2016-01-01'
```

Comment:
Can you add ` marks around the parameter name? 
```
The `date` parameter value...
```

Just need the first one since it's referring to the actual "date" parameter and not a date in general

+1 otherwise